### DISCUSSION TITLE
Quark easy transerring

### DISCUSSION THREAD

**Discussion by Marius-jj** [P3pp3rF1y/SophisticatedBackpacks#1194] [Ideas]
Would it be possible to add compatibility with the quark easy transferring buttons? So that you can insert/extract to/from the backpack to/from inventory and chests?

<img width="731" alt="Screenshot 2024-11-06 161855" src="https://github.com/user-attachments/assets/98ce279f-4598-44b6-a9a6-547633e68693">

**Comment by P3pp3rF1y** [comment]
I already had that in the past. realistically I had to reimplement the whole thing in sophisticated and then when screen was shown remove the quark button and replace it with a button that looked the same but that could trigger the sophisticated version of the functionality. Based on that my plan is to actually include this functionality natively in sophisticated mods because that's less work than adding "compatibility" which has to do more work to cover the fact that it's something fully implemented in the mod.

**Reply by Marius-jj** [reply]
Ahh okay! Good to hear it's a planned feature tho, makes life much easier haha! Great mod btw, can't play without it!
